static/js/main.js

```diff
@@ -123,7 +121,11 @@ function writeSettings() {
     }
   }).then((response) => response.json())
   .then((data) => {
-      console.log(data);
+    if (data.Error) {
```

Comment:
I'm not 100% sure but shouldn't this be in the catch block? Did you test if it works?